sklearn/metrics/cluster/_supervised.py

```diff
@@ -113,14 +108,20 @@ def contingency_matrix(labels_true, labels_pred, *, eps=None, sparse=False):
 
         .. versionadded:: 0.18
 
+    dtype : numeric data type, optional.
+        Defaults to np.int, but see the notes on ``eps`` below.
```

Comment:
Following the [guidelines for writing documentation](https://scikit-learn.org/stable/developers/contributing.html#guidelines-for-writing-documentation)
```suggestion
    dtype : numeric data type, default=np.int.
        See the notes on ``eps`` below.
```